### ISSUE TITLE
Test::Alien::Diag print warning in case when variables contain undef

### ISSUE THREAD

**Issue by michal-josef-spacek** [PerlAlien/Alien-Build#363]
I created Alien::bc::GNU, which deliver bc tool and when I add diag like:
```
use strict;
use warnings;

use Alien::bc::GNU;
use Test::Alien::Diag;
use Test::More 'tests' => 1;
use Test::NoWarnings;

# Test.
alien_diag('Alien::bc::GNU');
```

Output with Alien::Build@0.66 is:
```
1..1
#
#
#
# Alien::bc::GNU->cflags        =
# Alien::bc::GNU->cflags_static =
# Alien::bc::GNU->libs          =
# Alien::bc::GNU->libs_static   =
# Alien::bc::GNU->version       = 1.07.1
# Alien::bc::GNU->install_type  = share
# Alien::bc::GNU->bin_dir       = /home/skim/perl5/lib/perl5/x86_64-linux-gnu-thread-multi/auto/share/dist/Alien-bc-GNU/bin
#
#
not ok 1 - no warnings
#   Failed test 'no warnings'
#   at /home/skim/perl5/lib/perl5/Test/Builder.pm line 193.
# There were 4 warning(s)
#     Previous test 0 ''
#     Use of uninitialized value in sprintf at /home/skim/perl5/lib/perl5/Test/Alien/Diag.pm line 45.
#  at /home/skim/perl5/lib/perl5/Test/Alien/Diag.pm line 45.
#       Test::Alien::Diag::alien_diag("Alien::bc::GNU") called at 03-diag.t line 10
#
# ----------
#     Previous test 0 ''
#     Use of uninitialized value in sprintf at /home/skim/perl5/lib/perl5/Test/Alien/Diag.pm line 45.
#  at /home/skim/perl5/lib/perl5/Test/Alien/Diag.pm line 45.
#       Test::Alien::Diag::alien_diag("Alien::bc::GNU") called at 03-diag.t line 10
#
# ----------
#     Previous test 0 ''
#     Use of uninitialized value in sprintf at /home/skim/perl5/lib/perl5/Test/Alien/Diag.pm line 45.
#  at /home/skim/perl5/lib/perl5/Test/Alien/Diag.pm line 45.
#       Test::Alien::Diag::alien_diag("Alien::bc::GNU") called at 03-diag.t line 10
#
# ----------
#     Previous test 0 ''
#     Use of uninitialized value in sprintf at /home/skim/perl5/lib/perl5/Test/Alien/Diag.pm line 45.
#  at /home/skim/perl5/lib/perl5/Test/Alien/Diag.pm line 45.
#       Test::Alien::Diag::alien_diag("Alien::bc::GNU") called at 03-diag.t line 10
#
# Looks like you failed 1 test of 1.
```

**Comment by plicease** [comment]
This should be addressed in 2.67.  Thank you for reporting this.

**Comment by michal-josef-spacek** [comment]
@plicease Great, thank you.
